tests/acceptance-tests/wayland-generator/protocol.xml

```diff
@@ -2414,7 +2414,7 @@
     </event>
   </interface>
 
-  <interface name="wl_output" version="3">
+  <interface name="wl_output" version="4">
```

Comment:
It isn't strictly necessary to update the acceptance test input and reference output. But :shrug: